task6/src/services/userService.js

```diff
@@ -64,4 +64,13 @@ export default class UserService {
             ]
         });
     }
+
+    getUserByNameAndPassword(username, password) {
+        return this.model.findOne({
+            where: {
+                [Op.and]: [{ login: username, password }]
```

Comment:
Usually password is not stored in DB as is, but is encrypted. So you'll search for login & then compare passwords. e.g. with bcrypt npm.

Reply:
Thanks, I'll think about it